packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -42,13 +42,25 @@ typedef RefreshCallback = Future<void> Function();
 
 // The state machine moves through these modes only when the scrollable
 // identified by scrollableKey has been scrolled to its min or max limit.
-enum _RefreshIndicatorMode {
-  drag,     // Pointer is down.
-  armed,    // Dragged far enough that an up event will run the onRefresh callback.
-  snap,     // Animating to the indicator's final "displacement".
-  refresh,  // Running the refresh callback.
-  done,     // Animating the indicator's fade-out after refreshing.
-  canceled, // Animating the indicator's fade-out after not arming.
+/// Indicates current status of Material `RefreshIndicator`
```

Comment:
I'm not sure what the private comment means:
```
// The state machine moves through these modes only when the scrollable
// identified by scrollableKey has been scrolled to its min or max limit.
```

- There is no `scrollableKey` in the file. 
- I think this comment shouldn't be here because it specifically talks about the behavior of RefreshIndicator, but the enum is outside the class, and might be used for different purposes other than RefreshIndicator. Tell me if this makes sense to you.

Reply:
The enum wouldn't be used outside of the file because it's private, so it's likely an outdated comment. Looks like an old pattern was to name the key provided to a `Scrollable` as `scrollableKey`, but now we use a NotificationListener to do what I think `scrollableKey` would have been doing before.

I'd say remove the private comment.

Reply:
Will do! But the enum's name doesn't start with an underscore, why is it private?

Reply:
It was private before this PR as `_RefreshIndicatorMode`

Reply:
> It was private before this PR as `_RefreshIndicatorMode`

Should we make it private again? Asking this because I'm not the only one who worked on this PR, and these changes have been made by another user.

Reply:
No, it will be useful for it to be public so developers can put in custom behavior that watches for these states. Since it was private before we don't have to worry about a breaking change outside of this file though for this PR, which is nice.

Reply:
Right, haven't thought about that.